### DISCUSSION TITLE
Plugin Admin Config fields documentation

### DISCUSSION THREAD

**Discussion by LaocheXe** [e107inc/e107#5024] [Q&A Plugins] [answered]
I've notice there are a few fields that have been added, that isn't in the documentation. Or they have been there but never added to the doc.

In the plugin builder you can add Batch (Have no idea what this is used for, in the admin_config it's 'batch' => 'value'), Filter (guessing the filter the value, but shows as 'filter' => 'value', Validate ('validate'=>'value' - has the tool tip of (Require Field)), Display (have no clue where that is in admin_config (Tool Tip for this one is Display by Default)), R/O Read Only ('readonly'=>'value').

I want to know what they are, and add them to the Developer Document for the rest. Readonly, validate, and display have tool tips in plugin builder, but what do you place in the 'value' spot? When selecting all of them, they all have => 'value' - even inline, which should be true or false? If someone can give me a run down?

**Comment by Moc** [comment, marked as answer]
Documentation is very outdated. This one contains the most up-to-date information but is not finished:

https://devguide.e107.org/plugin-development/admin-ui

And it looks like the plugin builder needs some updates.
